doc/sphinx-guides/source/developers/windows.rst

```diff
@@ -16,3 +16,91 @@ See the `post <https://groups.google.com/g/dataverse-dev/c/utqkZ7gYsf4/m/4IDtsvK
+
+.. code-block:: powershell
+
+  wsl --install -d <Distribution Name>
+
+You will be asked to create a linux user.
+After installation of Linux check that you have internet connection:
+
+.. code-block:: bash
+
+  ping www.google.com
+
+If you do not have internet connection try add in ``/etc/wsl.conf``
+
+.. code-block:: bash
+  
+  [network]
+  generateResolvConf = false
+
+Also in /etc/resolv.conf add
+
+.. code-block:: bash
+
+  nameserver 1.1.1.1
+
+Now you can install all the tools one usually uses in Linux. For example, it is good idea to run update:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt full-upgrade -y
+
+Install Dataverse
+~~~~~~~~~~~~~~~~~
+
+Now you can install Dataverse in WSL following the instructions for :doc:`classic-dev-env`
+At the end check that you have ``-Ddataverse.pid.default-provider=fake`` in jvm-options.
+
+Now you can access dataverse in your windows browser
+
+ - http://localhost:8080
+ - username: dataverseAdmin
+ - password: admin
+
+IDE for Dataverse in Windows
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Files in WSL are accessible from Windows for editing using ``\\wsl.localhost`` or ``\\wsl$`` path. Windows files are accessible in linux in ``/mnt/c/`` directory. Therefore one can use one's favorite editor or IDE to edit dataverse project files. Then one can build using ``mvn`` in WSL and deploy manually in WSL using ``asadmin``.
+
+It is still though possible to use full strength of IDE, the following instructions are for Intelij users.
+
+- Install Intelij in Windows.
+
+You can open the project through ``\\wsl.localhost`` and navigate to dataverse project.
+You can try to build the project in Intelij. You may get a message ``Cannot establish network connection from WSL to Windows host (could be blocked by firewall).`` In that case you can try
+to disable WSL Hyperviser from firewall.
+After that you should be able to build the project in Intelij.
+It seems that at present it is impossible to deploy the glassfish application in Intelij. You can try to add Glassfish plugin through Settings->Plugins and in Run->Edit Configurations configure Application Server from WSL ``/usr/localhost/payara6`` with URL http://localhost:8080 and Server Domain as domian one, but it may fail since Intelij confuses the Windows and Linux paths.
+
+To use the full strength of Intelij with build, deployment and debugging, one will need to use Intelij ``Remote development``. Close all the projects in InteliJ and go to ``Remote development->WSL`` and press ``New Project``. In WSL instance choose your linux distribution and press ``Next``. In ``Prpject Directory`` navigate to WSL dataverse project.Then press ``Download IDE and Connect``. This will install InteliJ in WSL in ``~/.cache/JetBrains/``. Now in InteliJ you should see your project opened in a new InteliJ window. After adding Glassfish plugin and editing configuration you should be able to build the project and run the project.
```

Comment:
```suggestion
To use the full strength of Intelij with build, deployment and debugging, one will need to use Intelij ``Remote development``. Close all the projects in Intellij and go to ``Remote development->WSL`` and press ``New Project``. In WSL instance choose your Linux distribution and press ``Next``. In ``Prpject Directory`` navigate to WSL Dataverse project. Then press ``Download IDE and Connect``. This will install Intellij in WSL in ``~/.cache/JetBrains/``. Now in Intellij you should see your project opened in a new Intellij window. After adding Glassfish plugin and editing configuration you should be able to build the project and run the project.
```